packages/web/components/controls/QuestionRecorder.tsx

```diff
@@ -0,0 +1,369 @@
+"use client";
+import React, { useCallback, useEffect, useRef, useState } from "react";
+import BackendService from "@/services/BackendService";
```

Comment:
nit: Using `import type` is preferable for tree-shaking resp. resulting bundle sizes as it informs the bundler, i.e. webpack, that the import is only used for the type system but not at runtime.

```suggestion
import type BackendService from "@/services/BackendService";
```

No need to adapt this here, I'll think about introducing [consistent-type-imports linting rule](https://typescript-eslint.io/rules/consistent-type-imports/) in the future.